src/calculateRentalCost.js

```diff
@@ -21,14 +21,20 @@
  *  - calculateRentalCost(1) === 40
  *  - calculateRentalCost(3) === 100
  *  - calculateRentalCost(7) === 230
- *
- * @param {number} days
- *
- * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3 && days <= 6) {
+    return days * 40 - 20;
+  }
+
+  if (days < 3) {
```

Comment:
You can remove this condition `days < 3` and use just `return days * 40;` as the default case. 🙌

Reply:
Done